queries/cmake/indents.scm

```diff
@@ -0,0 +1,21 @@
+[
+  (normal_command)
+  (if_condition)
+  (foreach_loop)
+  (while_loop)
+  (function_def)
+  (macro_def)
+  (block_def)
+] @indent.begin
+
+[
+ (elseif_command)
+ (elseif_command)
+ (endif_command)
+ (endforeach_command)
+ (endfunction_command)
+ (endmacro_command)
+ (endblock_command)
+] @indent.branch
+
+(normal_command ")" @indent.branch)
```

Comment:
Maybe a `(normal_command ")" @indent.end)` after this to ensure it doesn't continue the previous indent level

Reply:
already added